nautobot/extras/forms/mixins.py

```diff
@@ -370,6 +369,12 @@ def _append_relationships(self):
                 field_name = f"cr_{relationship.slug}__{peer_side}"
                 self.fields[field_name] = relationship.to_form_field(side=side)
 
+                # if relationship.required in [
+                #     RelationshipSidesRequiredChoices.BOTH_SIDES_REQUIRED,
+                #     RelationshipSidesRequiredChoices.DESTINATION_SIDE_REQUIRED,
+                # ]:
+                #     self.fields[field_name].required = True
+
```

Comment:
Intended?

Reply:
Yeah it's commented out as intending to put the HTML5 validation back in when backend is sorted..